services_app/src/androidTest/java/org/opendatakit/database/service/GroupsPermissionTest.java

```diff
@@ -601,7 +602,7 @@ public void testUserAuthorizationFailure() {
          assertEquals(1, table.getNumberOfRows());
          row = table.getRowAtIndex(0);
 
-         assertEquals(2L,row.getDataByKey(COL_INTEGER_ID));
+         assertEquals(Long.valueOf(2),row.getDataByKey(COL_INTEGER_ID));
```

Comment:
This might make more sense being a constant? What do you think? So it's clear in the test when the value is being populated and when the value is being checked they are the same?

Reply:
Yes, I do think so too. I have created constants COL_INTEGER_VALUE_TWO, COL_INTEGER_VALUE_THREE, and COL_INTEGER_VALUE_FOUR that stores int 2, 3, and 4 respectively. This way, we can reuse the values and quickly identify its usages.